server/i18n/en.json

```diff
@@ -4896,23 +4896,23 @@
   },
   {
     "id": "app.command_webhook.get.internal_error",
-    "translation": "Unable to get the webhook."
+    "translation": "Unable to get the webhook {{.hook_id}}"
```

Comment:
I wonder what the value of mirroring the input as part of the error is. Doesn't the caller already know the HookId? The same question applies to some of the other error messages.

Reply:
You are absolutely right. If caller has the data, then it should log the data.